ui/apps/platform/src/utils/analytics.ts

```diff
@@ -0,0 +1,11 @@
+export function analyticsIdentity(userId: string, traits = {}): void {
+    return window.analytics?.identify(userId, traits);
+}
+
+export function analyticsPageVisit(type: string, name: string, additionalProperties = {}): void {
+    return window.analytics?.page(type, name, additionalProperties);
```

Comment:
As we discussed on a call, is optional chaining needed for `.page.?(type, name, additionalProperties)` method call and so on, in case of function calls before the methods are added to `analytics` data structure?

Reply:
From my understanding it shouldn't be a concern. I don't think analytics is set on the window until its been fully initialized